### ISSUE TITLE
Url change when component is outside of any Livewire component

### ISSUE THREAD

**Issue by danielreales7** [livewire-ui/modal#86]
I have in a simple blade page: `Livewire.emit('openModal', 'modals.colaboradores')` 
When I open the modal the url automatic has changed to 
![image](https://user-images.githubusercontent.com/5383214/132645140-09121c72-8051-41b4-bbae-e396976499fc.png)

And for example if I call to Modal: `wire:click="$emit('openModal', 'planes.modals.editar', {{ json_encode(["plan" => $plan->id, "nuevoPlan" => false]) }})"` the url not change.

Why?

This is a problem because when I load the page I have this error:
![image](https://user-images.githubusercontent.com/5383214/132645513-a23a0e73-8b7f-4a04-ae38-c5f731227a63.png)

Is there any way to fix this?

Thank you so much!

**Comment by PhiloNL** [comment]
Do you happen to use pagination on the base page or one of these modals?

**Comment by danielreales7** [comment]
> Do you happen to use pagination on the base page or one of these modals?

Hi @PhiloNL, I was talking to Livewire developers yesterday and they told me it is a small bug. This occurs because I am calling the component from a simple blade page instead of a component. He told me that they would fix it in the next patch. 
I made some changes to make it work correctly and I modified the url from javascript.

However this does not happen when I call from a component to another component.

The use of pagination does not matter because, as I tell you, it works correctly for me in another component.

Thank you so much for your help!
